docs/blog/v1.md

```diff
@@ -0,0 +1,188 @@
+
+### remark-mdx
+
+`remark-mdx` is the syntactic extension for MDX in remark.  It provides the parsing functionality for
+MDX as a _remark plugin_.  It’s a bit meta.  With `remark-mdx` you can parse MDX documents
+and perform transformations before writing them back to MDX strings.  This allows tools like prettier
+to use the exact same parser as MDX core by installing the library.
+
+### Custom pragma
+
+With v1 we’ve moved away from using `MDXTag` and are using a custom pragma which wraps `React.createElement`.
+We decided to update this approach so the JSX output is more legible, lighterweight, and more customizable.
+**This means that we can add support for any library that speaks JSX!**
+
+Special thanks to [@christopherbiscardi](https://christopherbiscardi.com) for implementing this feature.
+
+[Read the blog post](/blog/custom-pragma)
+
+### 📚 Documentation
+
+**Good libraries need great docs**, so we’ve been working on adding content and improving the overall documentation
+website experience.
+
+*   [Search (thanks Algolia)](https://mobile.twitter.com/4lpine/status/1114270174096412672)
+*   [Guides](https://mdxjs.com)
+*   [Automatic deployment via ZEIT](https://zeit.co)
+*   [Custom Gatsby theme](https://gatsbyjs.org)
+*   Reorganization and misc updates
+
+Special thanks to [@jxnblk](https://jxnblk.com) and [@wooorm](https://wooorm.com) for their help improving the
+docs and updating the build tooling.
+
+## Breaking changes
+
+Unfortunately, we’ve had to introduce a few breaking changes, but we’ve written a migration guide which
+will walk you through the steps needed to upgrade.
+
+*   🚨`@mdx-js/tag` is replaced by `@mdx-js/react` and an `@mdx` pragma - [migration guide](/migrating/v1#pragma)
+*   MDXProvider now merges component contexts when nested - [migration guide](/migrating/v1#mdxprovider)
+*   React support now requires `>= 16.8` in `@mdx-js/react` - [migration guide](/migrating/v1#react)
+
+[Read the full v1 migration guide](/migrating/v1)
+
+## Deprecations
+
+Luckily, we only need to introduce one deprecation.  The `mdPlugins` and `hastPlugins` options have been renamed
+`remarkPlugins` and `rehypePlugins` respectively.  For the time being we will issue a warning when the old options
+are used.  In v2, the old options will be removed.
+
+## 📈 Growth
+
+A major release is always a good time to take a step back and reflect on what’s been happening in terms of growth
+and the greater community.  The ecosystem surrounding MDX has already grown larger than we ever dreamed.  We’ve also
+seen projects/products/application we never even imagined.
+
+### Ecosystem
+
+*   [Prettier](https://prettier.io)
+*   [Docz](https://docz.site)
+*   [MDX Deck](https://github.com/jxnblk/mdx-deck)
+*   [Next.js](https://nextjs.org)
+*   [Gatsby](https://gatsbyjs.org)
+*   [AST Explorer](https://astexplorer.net)
+*   [Vue support (alpha)](/vue)
+*   [Demoboard](https://frontarm.com/demoboard/)
+*   [Editors](/editors)
```

Comment:
The above list is a bit of a mix, they’re cool things, but what makes them be in this list? Vue support is very different from prettier!

Reply:
Yeah, I kinda arbitrarily added some seemingly relevant things that are new or built around the format 🤷‍♂️